test/test_kramdown_list.rb

```diff
@@ -1,6 +1,22 @@
 require 'test_helper'
 
 class TestKramdownList < Minitest::Test
+  # NOTE: kramdown automatically injects `id` attribute
```

Comment:
Check the kramdown version in which this feature is introduced.

Reply:
Probably, v0.7.0. https://kramdown.gettalong.org/news.html#kramdown-070-released